openscenario/map_fragment/include/map_fragment/geometry_utilities.hpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MAP_FRAGMENT__GEOMETRY_UTILITIES__HPP_
+#define MAP_FRAGMENT__GEOMETRY_UTILITIES__HPP_
```

Comment:
Good point. Perhaps we can change to one of:
- `geometric_operations.hpp`
- `geometric_aliases_and_operations.hpp` (longer but more accurate)

Reply:
If you feel that the latter name is more appropriate, then this file should be split into two header files. Splitting them into two files seems a bit excessive at this point, but since I expect there will be more aliases in the future, I don't see a problem with it.

Reply:
[Done](https://github.com/tier4/scenario_simulator_v2/pull/1108/commits/cffe343a0260bda7c17deb99adf64a9917334764).